packages/relay-signer/src/models/transactions.ts

```diff
@@ -89,3 +89,7 @@ export type PaginatedTransactionResponse = {
   items: RelayerTransaction[];
   next?: string;
 };
+
+export type TransactionDeleteResponse = {
```

Comment:
nit: maybe a better name would be `TransactionCancelResponse`?

Reply:
For intents it's actually delete (we delete it from the db)